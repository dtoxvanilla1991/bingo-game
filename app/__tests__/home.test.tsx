import { render } from "@testing-library/react";
import { describe } from "node:test";
import Home from "@/app/page";

describe("home page", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
