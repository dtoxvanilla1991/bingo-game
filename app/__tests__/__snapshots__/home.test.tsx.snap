// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-between p-24 bg-amber-200 text-amber-800"
  >
    <section>
      <h1
        class="text-4xl font-bold text-center"
      >
        Welcome to Bingo Game
      </h1>
      <p
        class="text-center text-xl mt-4"
      >
        Game in Progress...
      </p>
    </section>
  </main>
</div>
`;
